src/Site_Command.php

```diff
@@ -452,15 +694,44 @@ private function create_site_root() {
 		return true;
 	}
 
+	private function maybe_verify_remote_db_connection() {
+		if( 'db' !== $this->db_host ) {
+
+			// Docker needs special handling if we want to connect to host machine.
+			// The since we're inside the container and we want to access host machine,
+			// we would need to replace localhost with default gateway			
+			if( $this->db_host === '127.0.0.1' || $this->db_host === 'localhost' ) {
```

Comment:
Where is the remote db connection? Isn't this suppose to handle that? I mean here we are just looking for local host database no remote db connection.

Reply:
Also, here we can do ip white-listing by adding private and public ip check for mysql host If needed.

Reply:
@rahulsprajapati This part is a little complicated. In essence, we're creating a separate network on localhost, that is why, if you want to connect to a db installed on your local machine, you need to use the gateway IP of the said separate network.

Reply:
> Where is the remote db connection? Isn't this suppose to handle that? I mean here we are just looking for local host database no remote db connection

Whoops yes. Fixed